ninja-core/src/test/java/ninja/diagnostics/DiagnosticErrorRendererTest.java

```diff
@@ -97,5 +103,35 @@ public void buildAndRenderDiagnosticErrorWithHTMLEntities() throws Exception {
         Assert.assertThat(out, containsString("Attribute with &lt; &gt; &amp; entities"));
         Assert.assertThat(out, containsString("Exception message with &lt; &gt; &amp; entities"));
     }
+
+    /* testedClasses: ninja/diagnostics/DiagnosticErrorRenderer.java */
```

Comment:
This line could be changed for consistency to just show the classname ie. DiagnosticErrorRenderer, as it is in the other files here. This is the difference between an individual test downloaded from platform (which gets the pathname of the file, as here, and downloading all tests which results in "testedClasses" only showing the classname (I previously logged a bug regarding this difference, TG-4875)